src/main/java/org/spongepowered/api/item/recipe/Recipe.java

```diff
@@ -47,29 +44,10 @@
 public interface Recipe {
 
     /**
-     * Returns the list of item types that result when successful crafting of
-     * this Recipe is completed.
+     * Gets the result of this {@link Recipe}.
      *
-     * @return The resultant list of item types
+     * @return the results of this {@link Recipe}
      */
-    List<ItemType> getResultTypes();
-
-    /**
-     * Checks if the given {@link GridInventory} fits the required constraints
-     * to craft this Recipe.
-     *
-     * @param grid The ItemGrid to check for validity
-     * @return True if the given input matches this recipe's requirements
-     */
-    boolean isValid(GridInventory grid);
```

Comment:
@Maxqia There's a difference between binary compatibility and something being inherently broken. This is a binary breakage, anything relying on these methods (while they should be deprecated) would ultimately fail to work between API builds. I don't see the reason why this wouldn't be possible to retain for the sake of deprecation and uselessness (you can even print out a pretty printed log if you want)